account_invoice_report_ddt_group/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015 Nicola Malcontenti - Agile Business Group
+# © 2016 Andrea Cometa - Apulia Software
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
+
+{
+    'name': "Account invoice report grouped by DDT",
+    'version': '9.0.1.0.0',
+    'description': """
```

Comment:
Move this key in README, template of readme on https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst